### ISSUE TITLE
[Request] mendeley-desktop 

### ISSUE THREAD

**Issue by SiyuChen** [ScoopInstaller/Extras#8365]
### Information

**Name:**  Mendeley-desktop 

**Description:** With Mendeley Desktop, you can easily organize and search your personal library, annotate documents and cite as you write.

**Homepage:** https://www.mendeley.com/

**Download link(s):** https://www.mendeley.com/reference-management/mendeley-desktop

P.s. Mendeley-desktop is different from Mendeley Reference Manager (which has been included in Extras)

**Comment by rashil2000** [comment]
Isn't Mendeley Reference Manager the new (improved) Mendeley app?

**Comment by SiyuChen** [comment]
> Isn't Mendeley Reference Manager the new (improved) Mendeley app?

Not really. See https://www.researchgate.net/post/Mendeley_Desktop_vs_Mendeley_Reference_Manager. Overall, most people agree that Mendeley Desktop is more stable and works better than Mendeley Reference Manager, They are different. It is good to keep both versions

**Comment by rashil2000** [comment]
I see, thanks for the info
